validation-test/compiler_crashers_2_fixed/0069-sr3657.swift

```diff
@@ -1,5 +1,7 @@
 // RUN: %target-swift-frontend %s -emit-ir
 
+// https://github.com/apple/swift/issues/46442
+
```

Comment:
Typo in file name: the correct SR is 3<ins>8</ins>57 (see https://github.com/apple/swift/pull/7304)